controllers/articles.js

```diff
@@ -0,0 +1,195 @@
+import crypto from 'crypto';
+import models from '../models/index';
+
+const tempUser = '2dd9e22b-d19d-4501-a6b5-00bb4ebd9b3e'; // This line has to be deleted.
```

Comment:
I see that you are setting the static Id and the comment says that the line has to be deleted, It would be better if you delete it. Or provide full information of who to delete it.